scripts/sem_bleaching_map.py

```diff
@@ -311,137 +349,315 @@ def configure_sem_for_tile(self):
             sem_pxs = self.ebeam.pixelSize.value
             scale = (self.subpxs / sem_pxs[0], self.subpxs / sem_pxs[1])
             self.ebeam.scale.value = scale
-        self.ebeam.resolution.value = self.tile_shape # just a spot
+        self.ebeam.resolution.value = self.tile_shape  # just a spot
 
-    def calc_xy_pos(self, roi, pxs):
+    def configure_sem_for_survey(self, roi):
         """
-        Compute the X and Y positions of the ebeam
+        Configure the SEM for regular survey image
+        """
+        self.ebeam.dwellTime.value = self.dtsem
+        self.ebeam.scale.value = (1, 1)
+
+        sem_pxs = self.ebeam.pixelSize.value
+        ccd_pxs = self.get_ccd_pxs()
+
+        full_width = self.ebeam.resolution.range[1]
+
+        # calculate the width of the border in rel coordinates
+        b_rel = [BORDER * ccd_pxs[0] / (full_width[0] * sem_pxs[0]),
+                 BORDER * ccd_pxs[1] / (full_width[1] * sem_pxs[1])]
+
+        self.ccd.fanSpeed.value = 0
```

Comment:
Add a comment that this is turning off the cooling and fan to reduce vibrations.